PhpAmqpLib/Channel/AbstractChannel.php

```diff
@@ -129,6 +134,18 @@ public function getChannelId()
     {
         return $this->channel_id;
     }
+    
+    public function setBodySizeLimit($max_bytes)
+    {
+        $max_bytes = intval($max_bytes);
+        
+        if ( $max_bytes > 0 ) {
+            $this->body_size_max = $max_bytes;
+        }
+        else {
```

Comment:
Do you mean the else shouldn't be preceded by a newline? If so, also fixed; if not, please be more specific.